.github/workflows/scripts/e2e/gmc_install.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+USER_ID=$(whoami)
+LOG_PATH=/home/$(whoami)/logs
+MOUNT_DIR=${KIND_MOUNT_DIR:-"/home/$USER_ID/.cache/huggingface/hub"}
+TOKEN_DIR=${KIND_TOKEN_DIR:-"/home/$USER_ID/.cache/huggingface/token"}
+IMAGE_REPO=${OPEA_IMAGE_REPO:-""}
+VERSION=${VERSION:-"latest"}
+
+function install_gmc() {
+    PLATFORM=${1:-"xeon"}
+    # Make sure you have to use image tag $VERSION for microservice-connector installation
+    echo "install microservice-connector on $PLATFORM, using repo $IMAGE_REPO and tag $VERSION"
+    echo "using namespace $SYSTEM_NAMESPACE, $CHATQNA_NAMESPACE, $CODEGEN_NAMESPACE, $CODETRANS_NAMESPACE and $DOCSUM_NAMESPACE"
```

Comment:
Are these examples namespaces necessary?

Reply:
No. Good point. I will remove them. Thank you for catching.